sections/06_discussion.md

```diff
@@ -6,22 +6,10 @@ can be further pruned.*
 
 ### Evaluation
 
-*What are the challenges in evaluating deep learning models that are specific to
-this domain?  This can include a discussion of ROC versus precision-recall
-curves for the imbalanced classes often encountered in biomedical datasets.
-It could also mention alternative metrics that are used in specific sub-areas
-such as enrichment factors in virtual screening.  A lack of true gold standard
-data for some problems complicates both training and evaluation.  Confidence-
-weighted labels are valuable when available.*
-
-*Is progress in some biomedical areas slowed when new predictions (e.g. from
```

Comment:
I added it in italic in 46dccc0. I would suggest dropping it - I don't think I have much to say there beyond the obvious at this stage. Wonder what others think about this.